### ISSUE TITLE
Visualising the colormap - Mona Lisa

### ISSUE THREAD

**Issue by OceanWolf** [matplotlib/viscm#5]
Just seen the Scipy talk, and loved the graphic of Mona Lisa.  For me that gives a much better way to view the colourmap than the sample images given as we have a well known image that we can use for comparison.  At present we just see abstract shapes which (I think) our brains find more difficult to interpret and thus form a meaningful opinion on the colourmap.

So I would suggest adding the picture of the Mona Lisa (or similar) to the visualising tools output, perhaps also for the colour-blindness test as well than I think would really help us.  The brain afaik has a very good detection algorithm for faces, which makes it very easy for us to tell a good colormap from a bad one, and thus give a better opinion.

**Comment by njsmith** [comment]
I don't have a strong opinion either way really, because ideally to assess
a colormap you want to look at a few dozen images. We kept it down to three
to make it manageable for sticking up on the webpage and fitting in a
single window, but it's always going to be a pretty extreme compromise.
.
I will say that the reason we didn't put a face or a photo in in the first
place is exactly that people are so good at reading images :-). It's a
pretty different situation where you're trying to interpret some abstract
data surface versus trying to interpret a full 3d image with shading and
shadows etc. -- our extreme skill at the latter makes it a great way to
cheat when trying to get a general idea across quickly during a talk, but
I'm not particularly convinced that good colormaps for photos (which is
basically, any colormap that has a good lightness ramp) are necessarily the
same as good colormaps for visualizing arbitrary scalar fields.

**Comment by OceanWolf** [comment]
Ahh, good point about the lightness, I guess we look at that more when looking at real images... I have no idea about the different factors in making a good colormap for analysing scalar fields, though obviously lightness helps when it comes to printing in black and white...

For me the biggest problem for me with abstract shapes comes from their abstractness, that we have no base to compare it to, for example when looking at the galaxy formation video we saw more detail in some than in others despite all having "perceptual uniformity", and without a base to compare to, we cannot make a judgement on which features actually exist, and which ones result due to the colormap (like jet does frequently).

**Comment by kthyng** [comment]
To try to see what the "real" data looks like, IBM researchers in the past
compared the image in the colormap of interest with the image in grayscale:
http://www.research.ibm.com/people/l/lloydt/color/color.HTM

On Wed, Sep 2, 2015 at 6:16 PM, OceanWolf notifications@github.com wrote:

> Ahh, good point about the lightness, I guess we look at that more when
> looking at real images... I have no idea about the different factors in
> making a good colormap for analysing scalar fields, though obviously
> lightness helps when it comes to printing in black and white...
> 
> For me the biggest problem for me with abstract shapes comes from their
> abstractness, that we have no base to compare it to, for example when
> looking at the galaxy formation video we saw more detail in some than in
> others despite all having "perceptual uniformity", and without a base to
> compare to, we cannot make a judgement on which features actually exist,
> and which ones result due to the colormap (like jet does frequently).
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/matplotlib/viscm/issues/5#issuecomment-137271247.

## 

Kristen M. Thyng
Assistant Research Scientist
Department of Oceanography
Texas A&M University
Eller O&M 618
http://kristenthyng.com

**Comment by njsmith** [comment]
We now have Kovesi's colormap test image, which is IMO better than any of the alternatives discussed here, so I think we can close this.
